packages/neos-ui/src/index.js

```diff
@@ -139,6 +134,11 @@ function * application() {
     const configuration = yield system.getConfiguration;
 
     //
+    // Bootstrap the saga middleware with initial sagas
+    //
+    allSagas.forEach(saga => sagaMiddleWare.run(saga, {store, globalRegistry, configuration}));
```

Comment:
that's I think too far below; as we cannot listen to actions.System.ready() anymore in sagas f.e.